frontend/src/components/WofInfoBox.vue

```diff
@@ -0,0 +1,117 @@
+<template>
+    <wof-popper :isOpen="this.isOpen">
+    <div :class="`${this.infoBoxType}`" v-if="this.isOpen">
+        <div class="info_box__title-close-container">
+            <div class="info_box__title">
+                {{ title }}
+            </div>
+            <div class="info_box__close-window-block">
+                <wof-button :variant="this.infoBoxType" :id="`info_box__close-button`" @click="$emit('close')">
+                    <wof-icon icon="wof-cancel" :size="2"  />
```

Comment:
You should not use icon in button like that. WofButton already has an icon inside. 
Remove this button and use instead WofIcon alone with click event like you did for button.
Use flexbox for "info_box__close-window-block" to position it properly and give 0.2em padding.
Add proper hover styles which will change font color for this appropriately